tests/tensorflow/nightly/transformer-translate.libsonnet

```diff
@@ -39,6 +39,11 @@ local gpus = import "templates/gpus.libsonnet";
       "--train_steps=20000",
     ],
   },
+  local functional_10k_steps = mixins.Functional {
```

Comment:
nit: Can you make this called `functional_short` or something like that?

Reply:
done